magento/adobe-stock-integration
# Do not show excluded directories and images in the media gallery

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1014: Do not show excluded directories and images in the media gallery
2. ...

### Manual testing scenarios (*)
Install magneto and adobe stock integration

1. 
2.     Login to Admin panel
3.     Go to Catalog -> Category
4.     Select any category
5.     Expand Content fieldset
6.     Click "Select from Gallery" button next to "Category Image" field
7.     Observer the media gallery directory tree

Excluded directories like captcha, downloadable, catalog/product etc. should not be displayed in the directory tree, the files from the excluded directories should not be present in displayed media assets
